front/app/modules/commercial/native_surveys/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactNode } from 'react';
+import { ModuleConfiguration } from 'utils/moduleUtils';
+import Tab from './admin/components/Tab';
+import { isNilOrError } from 'utils/helperUtils';
+import { IProjectData } from 'services/projects';
+import { IPhaseData } from 'services/phases';
+
+const AdminProjectIdeaEditFormComponent = React.lazy(
+  () => import('./admin/containers/projects/surveys')
+);
+
+type RenderOnHideTabConditionProps = {
+  project: IProjectData;
+  phases: IPhaseData[] | null;
+  children: ReactNode;
+};
+
+const RenderOnHideTabCondition = (props: RenderOnHideTabConditionProps) => {
+  const { project, phases, children } = props;
+  const processType = project.attributes.process_type;
+  const participationMethod = project.attributes.participation_method;
+  const noNativeSurveyInTimeline =
+    !isNilOrError(phases) &&
+    !phases.some(
+      (phase) => phase.attributes.participation_method === 'native_survey'
+    );
+  const hideTab =
```

Comment:
Here I'd just add a comment or two about the desired behavior ("hide the tab if it's a continuous project with no native survey") or something similar